tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -37,7 +56,13 @@ public function set_up() {
 		parent::set_up();
 
 		$this->context                = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
```

Comment:
Sorry for not pointing this out earlier, but it looks like a class property isn't required for the context and is not used outside the constructor. A simple local variable should be sufficient.